### PR TITLE
add mockinbird

### PR DESCRIPTION
**Pull request by croth1**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Let's see if this works 🎉

### CODE REVIEW

**Comment by croth1** [comment]
@bioconda/core should be ready for review.

File: recipes/mockinbird/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - pandas
+    - matplotlib
+    - seaborn
+    - pysam
+    - star
+    - bowtie
+    - samtools
+    - fastqc
+    - xxmotif
+    - r
```
Comment (bgruening) [inline recipes/mockinbird/meta.yaml]: This should be r-base I guess.

Comment (croth1) [reply recipes/mockinbird/meta.yaml]: true - fixed 👍

**Comment by croth1** [comment]
@bgruening, any more remarks? 👍

**Comment by bgruening** [comment]
Only one remark: Thanks!

**Pull request merged**